packages/base-styles/_mixins.scss

```diff
@@ -18,25 +18,25 @@
 @mixin heading-small() {
 	@include _text-heading();
 	font-size: $font-size-x-small;
-	line-height: $line-height-x-small;
+	line-height: $font-line-height-x-small;
```

Comment:
Ah, nice catch. It looks like these weren't working since https://github.com/WordPress/gutenberg/pull/65418 ?